getting-started/templates/GCP/gcp-supplemental-values.yaml

```diff
@@ -0,0 +1,381 @@
+      projected:
+        sources:
+          - serviceAccountToken:
+              audience: <audience-name> # <ATTENTION> - Enter the audience name for the projected service account token
+              expirationSeconds: 3600
+              path: token
+    # This volume is required for the proxy to authenticate with cloudSQL using a service account key file. 
+    - name: <config-volume-name> # <ATTENTION> - Enter the volume name where config.json is to be mounted
+      secret:
+        secretName: <secret-name> # <ATTENTION> - Enter the secret name where config.json is added.
+
+  connectionInfo:
+    ## @param database.connectionInfo.host PostgreSQL hostname. Since the cloud-sql-auth-proxy is used, this value should be localhost.
+    ##
+    host: "localhost"
+    ## @param database.connectionInfo.port PostgreSQL port
+    ##
+    port: "5432"
+    ## @param database.connectionInfo.dbName PostgreSQL database name
+    ##
+    dbName: <database-name> # <ATTENTION> - Enter the PostgreSQL database name for testmonitor service
+    ## @param database.connectionInfo.user PostgreSQL username used by the service
+    ##
+    user: <username> # <ATTENTION> - Enter the PostgreSQL username for the testmonitor service
+    ## @param database.connectionInfo.secretName The name of an existing secret with
+    ## PostgreSQL connection credentials
+    ##
+    secretName: <secret-name> # <ATTENTION> - Enter the secret name that contains the testmonitor service's Postgres SQL database connection credentials
+    ## @param database.connectionInfo.passwordKey Password key for database.connectionInfo.user to be retrieved from existing secret
+    ## NOTE: Ignored unless `database.connectionInfo.secretName` parameter is set.
+    ##
+    # passwordKey: "passkey"
+    ## @param database.connectionInfo.migrationPasswordKey Password key for database.connectionInfo.migrationUser to be retrieved from existing secret
+    ## If unset database.connectionInfo.passwordKey is used instead.
+    ## NOTE: Ignored unless `database.connectionInfo.secretName` parameter is set.
+    ##
+    # migrationPasswordKey: null
+
+  serviceAccount:
+    ## @param serviceAccount.annotations Annotations to add to the service account
+    ##
+    annotations:
+      {
+        iam.gke.io/gcp-service-account=<YOUR-GSA-NAME>@<YOUR-GOOGLE-CLOUD-PROJECT>.iam.gserviceaccount.com, # <ATTENTION> - Enter the Google Service Account name created for Cloud SQL access and the Google Cloud Project name
+      }
+    ## @param serviceAccount.name The name of the service account to use.
+    ## If not set and create is true, a name is generated using the fullname template
+    ##
+    name: ""
+
+fileingestion:
+  s3:
+    port: 443
+    bucket: <name-of-gcs-bucket> # <ATTENTION> - Enter the name of the GCS bucket for fileingestion service
+    scheme: "https://"
+    host: "storage.googleapis.com"
+    region: "us-east1"
```

Comment:
this should be something customers should configure. It shouldn't be us-east1 by default

Reply:
Added placeholder